lib/koi/composable_components.rb

```diff
@@ -29,6 +29,17 @@ def self.components
       @@components.values
     end
 
+    # Default fallacbk section type when no initial section type
+    # is used in composition
+    def self.fallback_section_type
+      @@fallback_section_type ||= "body"
+    end
+
+    # Method to change initial section type
+    def self.set_fallback_section_type(section_type)
```

Comment:
Could we maybe change the definition it to this?

```
def self.fallback_section_type=(section_type)
   @@fallback_section_type = section_type
end
```

Then you can set the default like so 

```
Koi::ComposableContent.fallback_section_type = "medium"
```

No point having a `set_x` method in ruby when you can define setters yourself!

Reply:
Great suggestion, that makes much more sense!